packages/db/src/scripts/import-lsj.ts

```diff
@@ -43,9 +43,8 @@ function toMarkDown(raw: string): string {
       .replaceAll('</Level3>', '')
       .replaceAll('</Level4>', '')
       .replaceAll('<lb />', '')
-      .replaceAll(/<ref=".*">/g, '')
-      .replaceAll(/<ref='.*'>/g, '')
-      .replaceAll('</ref>', '')
+      .replaceAll(/<ref=".*?">(.*?)<\/ref>/g, '<span class="ref">$&</span>')
+      .replaceAll(/<ref='.*?'>(.*?)<\/ref>/g, '<span class="ref">$&</span>')
```

Comment:
I'm not sure how I'm going to make the jump from this to React portals. I'm sure I can figure it out, but if you have any hints/ideas to jump start me, that would be very helpful!

Reply:
My initial thought was something like the following:

1. Use standard DOM methods to insert a div immediately after the verse.
2. Create the portal to the inserted node
3. Add the JSX to the portal

I haven't proven this will work, so as a backup, we can always just create the DOM nodes manually. We may also need to make a separate component so that other state changes don't trigger rerenders, requiring us to reinsert the extra DOM nodes.